ghost/core/package.json

```diff
@@ -59,6 +59,7 @@
   "dependencies": {
     "@extractus/oembed-extractor": "3.2.1",
     "@sentry/node": "7.86.0",
+    "@sentry/profiling-node": "^1.2.6",
```

Comment:
I __think__ this contains native binaries. Sometimes these are a pain and might not be installed by Yarn (when they fail to build/download). We could move this to `optionalDependencies` and try-catch the require?

Reply:
Thanks — updated ✅